controllers/kustomization_controller.go

```diff
@@ -550,8 +550,8 @@ func (r *KustomizationReconciler) validate(ctx context.Context, kustomization ku
 	applyCtx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 
-	cmd := fmt.Sprintf("cd %s && kubectl apply -f %s.yaml --timeout=%s --dry-run=%s --cache-dir=/tmp",
-		dirPath, kustomization.GetUID(), kustomization.GetTimeout().String(), kustomization.Spec.Validation)
+	cmd := fmt.Sprintf("cd %s && kubectl apply -f %s.yaml --timeout=%s --dry-run=%s --cache-dir=/tmp --force=%t",
```

Comment:
This will fail if `server` validation is used because `--force` is not supported. I've added it intentionally so that the user can get some feedback:
> validation failed: error: --dry-run=server cannot be used with --force

Reply:
I propose we skip validation when force is enabled. We need to update the field description with this behaviour.

Reply:
That's what I thought first as well but considering that client-side validation still works I thought it would be useful to have it enabled just for that. Do you think it's not really useful in that situation?

Reply:
I think we should downgrade the server-side dry-run to client, when force is enabled.

Reply:
Yes, that makes sense.

Reply:
This is not described on the API field.

Reply:
Added. Thank you, @hiddeco!